source/_integrations/zha.markdown

```diff
@@ -404,7 +404,9 @@ To help resolve any kinks or compatibility problems, report bugs as issues with
 
 ### Knowing which devices are supported
 
-There is no official compatibility list of supported devices for the simple reason that practically all devices Zigbee Home Automation that are fully compliant with the standards and specifications as set by the [Zigbee Alliance](https://zigbeealliance.org) should technically be compatible with this ZHA integration. The fact remains, however, that some hardware manufacturers do not always fully comply with each set specification, which can cause a few devices to only partially work or not work at all with ZHA, but developers can create workarounds for such issues via a solution for 'ZHA exception and deviation handling' that this implementation features. See that section for more information.
+There is no official compatibility list of supported devices for the simple reason that practically all Zigbee devices that are fully compliant with the official standards should technically be compatible with this Zigbee implementation and work out-of-the-box with the ZHA integration. The official standards are set by the [Zigbee Alliance](https://zigbeealliance.org) in the [Zigbee Cluster Library Specification (ZCL) Revision 8](https://zigbeealliance.org/wp-content/uploads/2021/10/07-5123-08-Zigbee-Cluster-Library.pdf) and earlier revisions of their ZCL specifications.
```

Comment:
What does this add to the average end-user?

The current text IMHO is already covering it, without adding unneeded implementation details.

Reply:
I think it be good to mention "Zigbee Cluster Library Specification (ZCL) R8" for reference now that ZHA is fully compliant. How about if I instead just slightly rewrite the existing sentence to mention ZCL R8 in the "ZHA exception and deviation handling" section instead in relation to describing [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)?

That would only slightly change this existing sentence there:

The ZHA implementation in Home Assistant relies on a library called "[ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)" to resolve issues with Zigbee devices that do not fully conform with the Zigbee standards. The few devices that deviate from the **Zigbee specifications set by the [Zigbee Alliance](https://zigbeealliance.org)** may therefore require proper bug reports with debug logs from users to assistant the developers in writing custom [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) for all of a device functions to work properly with the ZHA integration.

Into:

The ZHA implementation in Home Assistant relies on a library called "[ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)" to resolve issues with Zigbee devices that do not fully conform with the Zigbee standards. The few devices that deviate from the **official standards set by the [Zigbee Alliance](https://zigbeealliance.org) in the [Zigbee Cluster Library Specification](https://zigbeealliance.org/wp-content/uploads/2021/10/07-5123-08-Zigbee-Cluster-Library.pdf) (revision 8 and earlier revisions of their ZCL specifications)** may therefore require proper bug reports with debug logs from users to assistant the developers in writing custom [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) for all of a device functions to work properly with the ZHA integration.

Reply:
> Zigbee Cluster Library Specification (ZCL) R8

No end-user cares about library specifications. They buy bulbs and want them to work.

Reply:
Closing PR